node/node.go

```diff
@@ -1767,7 +1767,8 @@ func (app *App) startAPIServices(ctx context.Context) error {
 			app.Config.API.JSONCorsAllowedOrigins,
 		)
 
-		if err := app.jsonAPIServer.StartService(ctx, maps.Values(publicSvcs)...); err != nil {
+		if err := app.jsonAPIServer.StartService(ctx, app.Config.CollectMetrics,
+			maps.Values(publicSvcs)...); err != nil {
```

Comment:
We are already providing the metrics on `app.Config.MetricsPort` why do we need to serve it on the JSON Service as well?